### DISCUSSION TITLE
Question on how remix loader handles external requests

### DISCUSSION THREAD

**Discussion by myeljoud** [remix-run/remix#9727] [Q&A] [answered]
Hello there,

I have this problem where when I do a `fetch` with a `POST` method to and external service I have from my `loader`, it seems that the request happens as a GET request, I'm confused because when I change the fetch method to anything other than `POST` the request gets executed as that method.

The API accepts only `POST` requests, mind you I can make the request from another client like Postman or Thunder Client, and it works fine but when making it from the `loader` I get "405 Method not allowed" even though the `method` is set to `POST`.

Is there something I'm missing on how Remix handles external requests inside loaders?

**Comment by sergiodxa** [comment, marked as answer]
If you could share more code on how you do it that would be helpful. Remix doesn't affect the way you use `fetch` in your app, so there may be something else happening.

**Comment by myeljoud** [comment]
Hello @sergiodxa,

Thanks so much for your response! It really boosted my confidence and helped me realize that the error might be from the API itself. Turns out, you were right—the API was redirecting my fetch from a `POST` to a `GET`, causing the error. Problem solved now!

Thanks a ton!
